src/drivers/express/ExpressAdminRouteDriver.ts

```diff
@@ -124,44 +122,16 @@ export class ExpressAdminRouteDriver {
     });
     router.patch(
       '/users/:username/learning-objects/:learningObjectName/publish',
-      async (req, res) => {
-
-        try {
-          const id = req.body.id;
-          const published = req.body.published;
-
-          await AdminLearningObjectInteractor.togglePublished(
-            this.dataStore,
-            req.params.username,
-            id,
-            published,
-          );
-          res.sendStatus(200);
-        } catch (e) {
-          console.error(e);
-          res.status(500).send(e);
-        }
+      async (_, res) => {
```

Comment:
Can you explain the _ instead of req? I have not seen this before.

Reply:
I picked this up a while back from a different project I was working on. Using an underscore to indicate that a variable is not used is sort of a semi-standard pattern. [This thread on SO](https://stackoverflow.com/questions/32197927/standard-conventions-for-indicating-a-function-argument-is-unused-in-javascript) talks a little about this.

The more I think about it though, I kind of like the idea of just calling it `req` and having my editor notify me that it is unused.

Reply:
I agree. I think it's a better option.

```suggestion
      async (req, res) => {
```